src/mlsquare/adapters/sklearn.py

```diff
@@ -2,10 +2,12 @@
 # -*- coding: utf-8 -*-
 from ..optmizers import get_best_model
 from ..utils.functions import _parse_params
+from ..architectures import sklearn
 import pickle
 import onnxmltools
 import numpy as np
 
+
```

Comment:
entire SklearnKerasRegressor is filled with isinstance checks. This should be avoided.